tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1271,6 +1268,34 @@ class MklConvOp : public OpKernel {
 
     AllocateOutputSetMklShape(context, kOutputIndex_Dst, output_tensor,
                               output_tf_shape, output_mkl_shape);
+    if (fuse_add_) {
+      const Tensor& add_tensor = MklGetInput(context, kInputIndex_Add);
+      MklDnnShape add_mkl_shape;
+      GetMklShape(context, kInputIndex_Add, &add_mkl_shape);
+
+      // Check if need reorder
+      if (!(add_mkl_shape == output_mkl_shape)) {
```

Comment:
This reads a bit funny. Since `MklDnnShape` doesn't overload `!=`, how about just putting the `add_mkl_shape == output_mkl_shape` case first?
```
if (add_mkl_shape == output_mkl_shape) {
  CHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
} else {
  auto add_md = 
     ...
}